src/delta_lake/delta_service.py

```diff
@@ -200,7 +200,9 @@ def _store_in_cache(
         )
 
 
-def count_delta_table(spark: SparkSession, database: str, table: str) -> int:
+def count_delta_table(
+    spark: SparkSession, database: str, table: str, use_cache: bool = True
```

Comment:
I think the docstrings need to be updated for all these functions

Reply:
👍

Reply:
:+1: